specification/2.0/schema/image.schema.json

```diff
@@ -29,5 +28,8 @@
     "additionalProperties": false,
     "dependencies": {
         "bufferView": [ "mimeType" ]
+    },
+    "not": {
```

Comment:
@bghgary @sbtron 
Given that the spec doesn't define a case when both fields aren't present, how about changing this to `"oneOf"`? (I.e. one of these fields must be provided.)
```json
{
  "oneOf": [
    {"required":["uri"]},
    {"required":["bufferView"]}
  ]
}
```

Otherwise we can add a note on how to handle a case when neither of them is provided.